src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -33,21 +44,47 @@
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService(); 
+        // add hard coded meals
+        // 0...4
+        datastore.put(createMealEntity(
+            new Meal(0L, "Fried potato", "Fried potato with mushrooms and onion.",
+                new ArrayList<>(Arrays.asList("potato", "onion", "mushrooms", "oil")), "Main")));
+        datastore.put(createMealEntity(
+            new Meal(1L, "Italian pizza", "Pizza with pineaple, sausage and tomato.",
+                new ArrayList<>(Arrays.asList("flour", "water", "sausage", "tomato", "pineaple", "cheese")), "Pizza")));
+        datastore.put(createMealEntity(
+            new Meal(2L, "Vegetable soup", "Vegetable soup with onion.",
+                new ArrayList<>(Arrays.asList("potato", "onion", "cabbage", "mushrooms", "water", "carrot", "pumpkin")), "Soup")));
+        datastore.put(createMealEntity(
+            new Meal(3L, "Chocolate cake", "Chocolate cake with butter cream and strawberry.",
+                new ArrayList<>(Arrays.asList("flour", "water", "butter", "cocoa powder", "chocolate", "sugar", "strawberry", "eggs")), "Dessert")));
+        datastore.put(createMealEntity(
+            new Meal(4L, "Hot chocolate", "Hot chocolate with sugar and caramel.",
+                new ArrayList<>(Arrays.asList("cocoa powder", "sugar", "milk", "caramel", "vanilla")), "Drinks")));
+        // 5...9
+        datastore.put(createMealEntity(
+            new Meal(5L, "Cheese Omelette", "This tasty omelette is full of cheese.",
+                new ArrayList<>(Arrays.asList("2 eggs", "salt", "butter", "Cheddar cheese", "Mozarella cheese")), "omelette")));
+        datastore.put(createMealEntity(
+            new Meal(6L, "Garlic, Tomato and Cheddar Sandwich", "A delicious sandwich for breakfast and fast food!",
+                new ArrayList<>(Arrays.asList("2 slices brown bread", "clove garlic", "tomato", "Cheddar cheese", "green chili",
+                "olive oil", "salt", "pepper")), "Main")));
+        datastore.put(createMealEntity(
+            new Meal(7L, "Chocolate Chips Pancake", "So delicious and chocolate breakfast.",
+                new ArrayList<>(Arrays.asList("2 eggs", "salt", "butter", "flour", "sugar", "chocolate chips", "milk")), "Pancake")));
+        datastore.put(createMealEntity(
+            new Meal(8L, "Seafood Pizza", "Just from the bottom of the ocean...",
+                new ArrayList<>(Arrays.asList("flour", "salt", "butter", "water", "oil", "shrimp", "tuna", "onion", "ctabstick",
+                "scallops", "fish sticks", "garlic", "tomato sauce")), "Pizza")));
+        datastore.put(createMealEntity(
+            new Meal(9L, "Almond Fudge Brownies", "There is any almond lover's prime recipe for fudge brownies.",
+                new ArrayList<>(Arrays.asList("2 eggs", "sugar", "butter", "milk", "almond", "vanilla", "cocoa powder")), "Dessert")));
```

Comment:
Please do not add entries to Datastore during startup.

Datastore is persistent storage that exists independently of your Java application. Once something is written to a Datastore table, it will still be there when you restart / update your app.
So don't add the hard-coded entries here. Instead use Cloud Console interface to manually add entries to your Datastore in `polinasand-step-2020` project.